pkgs/applications/virtualization/containerd/default.nix

```diff
@@ -14,19 +13,16 @@ stdenv.mkDerivation rec {
     sha256 = "07axhrvy45zwnsrvr2618227bzrnqdcjdl7j0mnrhvlryypiic0l";
   };
 
-  buildInputs = [ go libapparmor libseccomp ];
-
-  makeFlags = ''BUILDTAGS+=seccomp BUILDTAGS+=apparmor'';
+  buildInputs = [ makeWrapper go ];
```

Comment:
Is seccomp disabled after this commit? I don't think it's a good idea from a security standpoint.